lib/plugins/keypassPlugin.js

```diff
@@ -49,14 +49,14 @@ var keypassOperations = [
 function extractAction(req, res, callback) {
```

Comment:
It isn't even a fix, really. Is a preventive patch to avoid a problem we detected in previous PRs. A really, really minor refactor.